src/isar/mission_planner/mission_validator.py

```diff
@@ -0,0 +1,9 @@
+from typing import List
+
+from isar.models.mission import Mission
+
+
+def validate_robot_capabilities(
+    mission: Mission, robot_capabilities: List[str]
+) -> bool:
+    return all([task.type in robot_capabilities for task in mission.tasks])
```

Comment:
Suggestion: Fetch the robot capabilities here instead
```suggestion
def is_robot_capable_of_mission(mission: Mission) -> bool:
    robot_capabilities = robot_settings.CAPABILITIES
    return all([task.type in robot_capabilities for task in mission.tasks])
```